server/src/main/java/timely/api/model/Metric.java

```diff
@@ -109,7 +122,9 @@ public void setVisibility(ColumnVisibility visibility) {
 
     public Mutation toMutation() {
         final byte[] row = rowCoder.encode(new ComparablePair<String, Long>(this.metric, this.timestamp));
-        final Value value = new Value(valueCoder.encode(this.value));
+        WRITE_BUFFER.get().clear();
+        WRITE_BUFFER.get().putDouble(this.value);
+        final Value value = new Value(WRITE_BUFFER.get().array());
```

Comment:
Yeah, I had missed the fact that you're only ever putting doubles in there, so it's not an issue. It's still a safer practice to use the size, though in this case that would require more method calls.